README.md

```diff
@@ -114,6 +114,26 @@ Once you have your API key, you'll need to configure it in your environment. Use
 export MECHX_API_KEY=<your api key>
 ```
 
+### Set up agent mode for on-chain interactions (For Latest Marketplace Mechs only)
```

Comment:
I would say that we need to re-org this readme.

My suggestion is:
let's have all the MM from top to botton with agent mode (without specifying that it is agent mode but providing the context on what will happen). Then let's have a subsection of the MM one with private key set, then we have a final section with legacy description.

Reply:
additionally, we need to the following mention where funds needs to be transferred in case of agent mode / non agent mode for the various payment models